src/utilities/createRouterReject.ts

```diff
@@ -5,6 +5,22 @@ import { RegisteredRejectionType, Resolved, Route, RouteComponent } from '@/type
 export const builtInRejections = ['NotFound'] as const
 export type BuiltInRejectionType = typeof builtInRejections[number]
 
+const notFoundRoute = {
+  name: Math.random().toString(),
```

Comment:
the point here is that the name doesn't matter.. We could probably use a more elegant way of "random string"

Reply:
I feel pretty strongly we shouldn't use a random string for this because it is possible an end user could see this. Something specific like `NotFoundRejectionRoute` should be fine.